core/templates/dev/head/pages/collection_editor/CollectionEditorStateServiceSpec.js

```diff
@@ -95,6 +125,21 @@ describe('Collection editor state service', function() {
         acquired_skills: ['interest']
       }]
     };
+
+    fakeCollectionRightsBackendApiService.newBackendCollectionRightsObject = {
+      collection_id: '5',
+      can_edit: 'true',
+      can_unpublish: 'false',
+      is_private: 'true',
+      owner_names: ['A']
+    };
+    secondBackendCollectionRightsObject = {
```

Comment:
Can you come up with more descriptive names for ``newBackendCollectionRightsObject`` and ``secondBackendCollectionRightsObject``? It will help make your tests more readable and, in the future, if anyone is adding a new test it will be clearer to them which object to use (it currently isn't clear).